xml/System.IO.Compression/BrotliDecoder.xml

```diff
@@ -22,7 +22,7 @@
     </Interface>
   </Interfaces>
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Provides non-allocating, performant single-pass Brotli decompression methods that are independent from the <see cref="T:System.IO.Compression.Brotli.BrotliDecoder" /> instance.</summary>
```

Comment:
> `that are independent from the <see cref="T:System.IO.Compression.Brotli.BrotliDecoder" /> instance`

What does this mean?

Reply:
Gahh I made a mistake. Thanks for pointing it out. I meant `BrotliStream`, not `BrotliDecoder`.

I got this explanation from the [API proposal](https://github.com/dotnet/corefx/issues/25785), but shorter:

----
> ### BrotliEncoder & BrotliDecoder
> #### Proposed API
> The goal of the streamless implementation is to provide a non-allocating, performant Brotli implementation free from Streams. It contains simple Compress/Decompress operations that return an enum indicating the success of the operation as well as static CompressFully/DecompressFully operations that allow single-pass compression/decompression without the need for a BrotliEncoder/BrotliDecoder instance.
----

How about:
```suggestion
    <summary>Provides non-allocating, performant and single-pass Brotli decompression methods that are free from streams; that is, they do not depend on creating a <see cref="T:System.IO.Compression.Brotli.BrotliStream " /> instance.</summary>
```

Reply:
I don't the current summary is correctly phrased (for example, I wouldn't say "free from streams" - reading it from an end-user perspective, I would ask myself, what does this mean?). Taking snippets from the API proposal is exposing design discussion level detail in an API documentation/description which doesn't fit the audience of docs, imo.

When would I use this API over the ones on BrotliStream?

Reply:
> _I wouldn't say "free from streams" - reading it from an end-user perspective, I would ask myself, what does this mean?_

I think you're going to the other extreme, which is assuming a user that wants to use this API is completely ignorant/unfamiliar with streams. I think a user that visits a compression-related documentation page may already know a thing or two about using streams to compress/decompress.

> _When would I use this API over the ones on BrotliStream?_

Precisely! This API is intended to be used when the user does not want to depend on streams, that was the main reason why this API was proposed.

@tdykstra thanks for the correction, I will try to keep the sentences short. I need to find a way to explain the API more clearly, but in short sentences.

Reply:
I'll use @tdykstra 's suggestion, it makes sense, will contain a link for the user to click it and figure out why BrotliStream is not a dependency:

```suggestion
    <summary>Provides non-allocating, performant Brotli decompression methods. The methods decompress in a single pass without using a <see cref="T:System.IO.Compression.Brotli.BrotliStream" /> instance.</summary>
```